### PR TITLE
feat: [FFM-3943]: Basic integration

### PR DESCRIPTION
**Pull request by knagurski**
This PR adds basic basic integration with the FF UI MFE. No functionality should be effected.

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 2 Issues 
* Code issue 2 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11453)

**Review by laura-mannus — APPROVED** [review]
Brill!

File: configs/devServerProxy.config.js [hunk 1]
```diff
@@ -46,6 +46,10 @@ module.exports = {
   '/cv/api': {
     target: targetLocalHost ? 'https://localhost:6060' : `${baseUrl}`
   },
+  '/cf/web': {
+    pathRewrite: { '^/cf/web': '' },
+    target: process.env.FF_UI_URL || 'http://localhost:9191'
```
Comment (abhinavrastogi-harness) [inline configs/devServerProxy.config.js:51]: ET module also uses port 9191.. can we pick some other port for this service?

**Review by abhinavrastogi-harness — DISMISSED** [review]

File: configs/devServerProxy.config.js (hunk 1 above)
Comment (knagurski) [reply configs/devServerProxy.config.js]: 9292 it is 👍🏻

**Review by mtran7 — APPROVED** [review]

**Pull request merged**